src/layouts/BaseLayout/BaseLayout.jsx

```diff
@@ -49,9 +49,7 @@ function BaseLayout({ children }) {
               {isFullExperience ? (
                 children
               ) : (
-                <Suspense fallback={null}>
-                  <TermsOfService />
-                </Suspense>
```

Comment:
I think you should keep the `Suspense` boundary here as you are making a request in the `OnboardingOrChildren` component me thinks. Just peaking at `OnboardingOrChildren` you're returning a `Suspense` boundary which should be lifted up one level.